store/mockstore/unistore/cophandler/cop_handler.go

```diff
@@ -501,7 +501,7 @@ func genRespWithMPPExec(chunks []tipb.Chunk, lastRange *coprocessor.KeyRange, co
 		}
 	}
 	resp.ExecDetails = &kvrpcpb.ExecDetails{
-		TimeDetail: &kvrpcpb.TimeDetail{ProcessWallTimeMs: int64(dur / time.Millisecond)},
+		TimeDetail: &kvrpcpb.TimeDetail{ProcessWallTimeMs: uint64(dur / time.Millisecond)},
```

Comment:
To make compile ok: https://github.com/pingcap/kvproto/pull/931/files#diff-a546353419478c2fd5b495073d1eee706048b7524e9973f0aa846e0959c5ba42R893